src/Translation/Translator.php

```diff
@@ -59,8 +59,10 @@ public function get($key, array $replace = [], $locale = null, $fallback = true)
          *     });
          *
          */
-        if (isset($this->events) &&
-            ($line = $this->events->fire('translator.beforeResolve', [$key, $replace, $locale], true))) {
```

Comment:
This is our current style for handling multi-line conditionals, please go back to it